src/utils.rs

```diff
@@ -55,6 +56,35 @@ pub fn encode_with_code(output: &mut String, text: &str) {
     }
 }
 
+static UNICODE_CHARS_MAP: phf::Map<char, &str> = phf_map! {
+    '“' => "\"",
+    '”' => "\"",
+    '‘' => "\'",
+    '’' => "\'",
+    '—' => "--"
+};
+
+/// Normalize the mistakenly inputted Unicode character
+/// to the corresponding ASCII character.
+///
+/// For the table what characters this function will convert,
+/// you can refer to [`UNICODE_CHARS_MAP`].
+///
+/// Time complexity of this is `O(n)`.
+pub fn normalize_unicode_chars(inputs: &str) -> String {
```

Comment:
There is a single input string, so it should probably be `input` not its plural form.